src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/IntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.config;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * ## Integration test annotation.
+ * This annotation is meant for all the integration tests.
+ * Its purpose is to:
+ *  - centralise the integration test configuration
+ *  - avoid copy and past
+ *  - enable one profile for efficient running only @DirtysContext will restart/refresh the orchestrator context.
+ */
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles({"integration","nosb"}) // no servicebus queue handler registration
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
```

Comment:
Sounds dirty. Should we not use it until definitely necessary?

Reply:
removed